src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -1055,6 +1055,24 @@ CHIP_ERROR P256Keypair::Initialize(ECPKeyTarget key_target)
     return error;
 }
 
+CHIP_ERROR P256Keypair::ImportRawKeypair(const uint8_t * private_key, const size_t private_key_size, const uint8_t * public_key,
+                                         const size_t public_key_size)
+{
+    Crypto::P256SerializedKeypair serialized_keypair;
+    ReturnErrorOnFailure(serialized_keypair.SetLength(private_key_size + public_key_size));
+    memcpy(serialized_keypair.Bytes(), public_key, public_key_size);
+    memcpy(serialized_keypair.Bytes() + public_key_size, private_key, private_key_size);
+    return this->Deserialize(serialized_keypair);
+}
+
+CHIP_ERROR P256Keypair::ImportRawKeypair(const uint8_t * key_data, size_t key_data_size)
+{
+    Crypto::P256SerializedKeypair serialized_keypair;
+    memcpy(serialized_keypair.Bytes(), key_data, key_data_size);
```

Comment:
How do we know key_data_size is not too big here?  Shouldn't the SetLength (1) happen before this line and (2) be error-checked?